clang/test/dpct/helper_files_ref/include/util.hpp

```diff
@@ -15,6 +15,31 @@
 #include <cassert>
 #include <cstdint>
 
+// TODO: Remove these function definitions once they exist in the DPC++ compiler
```

Comment:
can those API works with computecpp? If not, we may need add macro to protect them

Reply:
can not work with computecpp